lib/puppetserver/ca/logger.rb

```diff
@@ -17,6 +17,12 @@ def level
         @level
       end
 
+      def debug?
```

Comment:
I don't think this method is necessary, since the `debug` method already checks for the debug level before logging.

Reply:
Yeah. I implemented because of how I used to implement debug mode for this command.  I can remove it.